cmd/copy.go

```diff
@@ -162,21 +162,21 @@ func (raw *rawCopyCmdArgs) parsePatterns(pattern string) (cookedPatterns []strin
 // to use fractions of a MiB. E.g.
 // 0.25 = 256 KiB
 // 0.015625 = 16 KiB
-func blockSizeInBytes(rawBlockSizeInMiB float64) (uint32, error) {
+func blockSizeInBytes(rawBlockSizeInMiB float64) (int64, error) {
```

Comment:
>int64 [](start = 50, length = 5)

shouldn't it be uint64? This number can't be negative

Reply:
Offline talk: It is better to have these as ints, because the blockSizes in sdk are ints.